server/commands/write.go

```diff
@@ -67,8 +68,10 @@ func (c *WriteCommand) validateTuplesets(ctx context.Context, req *openfgapb.Wri
 		}
 
 		// Validate that we are not trying to write to an indirect-only relationship
-		err = validateHasDirectRelationship(tupleUserset, tk)
-		if err != nil {
+		if err := validateHasDirectRelationship(tupleUserset, tk); err != nil {
```

Comment:
Under the new `typesystem` package there should be a method called `IsDirectlyAssignable(objectType, relation string)` which will return 'true' if the relation is assignable and 'false' otherwise.

We should use that here instead.

Reply:
Cool, updated.